docs/user-guide/src/version_support.md

```diff
@@ -31,15 +29,11 @@ Following the same trend, the following tags for BMO and Ironic are available an
 - capm3-v1.1.2 (BMO and Ironic)
 - capm3-v1.1.1
 - capm3-v1.1.0
```

Comment:
Some newer tags are missing here also, check here https://quay.io/repository/metal3-io/ironic?tab=tags

Reply:
Fixed in #330